### PR TITLE
Added portfolio works and about section

### PR DESCRIPTION
**Pull request by ONJoseph**
- Updated the README file
- Added new images to the image folder
- added the work and about section code to index.html file
- added the works and about section to the style.css file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -56,6 +65,152 @@ <h1 class="head__title fz-40 fw-700 lh-52">I'm Joseph Glad to see you!</h1>
               </ul>
           </div>
         </section>
-      </div>
+        
+	  <section class="works">
```
Comment (OLIPLICHE) [inline index.html:69]: - [x] Please make sure that in this `works section`, the main container of the projects has a `display:grid` 👍

File: index.html [hunk 2]
```diff
@@ -56,6 +65,152 @@ <h1 class="head__title fz-40 fw-700 lh-52">I'm Joseph Glad to see you!</h1>
...
+						  <li class="works__card__subtitle-li fw-600 fz-13 lh-16 clr-n-500">Back End Dev</li>
+						  <li class="works__card__subtitle-li fw-600 fz-13 lh-16 clr-n-500">2015</li>
+					  </ul>
+				  </div>
+				  <p class="works__card__dsc fw-400 fz-15 lh-24 clr-n-600">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+				  <div class="works__card__topics flex">
+					  <p class="works__card__topics__text fw-500 fz-12 lh-16 clr-p-400 bg-p-50">html</p>
+					  <p class="works__card__topics__text fw-500 fz-12 lh-16 clr-p-400 bg-p-50">css</p>
+					  <p class="works__card__topics__text fw-500 fz-12 lh-16 clr-p-400 bg-p-50">javascript</p>
+				  </div>
```
Comment (OLIPLICHE) [inline index.html]: - [x] Please make sure the list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `<ul>` and each of them a `<li>` 👍 .

File: index.html [hunk 3]
```diff
@@ -56,6 +65,152 @@ <h1 class="head__title fz-40 fw-700 lh-52">I'm Joseph Glad to see you!</h1>
...
+						<li class="works__card__subtitle-li fw-600 fz-13 lh-16 clr-n-500">Back End Dev</li>
+						<li class="works__card__subtitle-li fw-600 fz-13 lh-16 clr-n-500">2015</li>
+					</ul>
+				</div>
+				<p class="works__card__dsc fw-400 fz-15 lh-24 clr-n-600">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+				<div class="works__card__topics flex">
+					<p class="works__card__topics__text fw-500 fz-12 lh-16 clr-p-400 bg-p-50">html</p>
+					<p class="works__card__topics__text fw-500 fz-12 lh-16 clr-p-400 bg-p-50">css</p>
+					<p class="works__card__topics__text fw-500 fz-12 lh-16 clr-p-400 bg-p-50">javascript</p>
+				</div>
```
Comment (OLIPLICHE) [inline index.html]: - [x] Please make sure the list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `<ul>` and each of them a `<li>` 👍 .

File: index.html [hunk 4]
```diff
@@ -56,6 +65,152 @@ <h1 class="head__title fz-40 fw-700 lh-52">I'm Joseph Glad to see you!</h1>
...
```
Comment (OLIPLICHE) [inline index.html]: - [x] Please make sure the list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `<ul>` and each of them a `<li>` 👍 .

File: index.html [hunk 5]
```diff
@@ -56,6 +65,152 @@ <h1 class="head__title fz-40 fw-700 lh-52">I'm Joseph Glad to see you!</h1>
...
```
Comment (OLIPLICHE) [inline index.html]: - [x] Please make sure the list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `<ul>` and each of them a `<li>` 👍 .

File: index.html [hunk 6]
```diff
@@ -56,6 +65,152 @@ <h1 class="head__title fz-40 fw-700 lh-52">I'm Joseph Glad to see you!</h1>
...
+				<img src="images/multi-post-project.png" alt="multi-post stories project">
+			</div>
+			<div class="works__card__text">
+				<h2 class="works__card__title fw-700 fz-32 lh-44 clr-n-800">Multi-Post Stories</h2>
+				<div class="works__card__subtitle flex">
+					<p class="works__card__subtitle-1 fw-600 fz-13 lh-16 clr-n-600">CANOPY</p>
+					<ul class="works__card__subtitle-ul flex">
+						<li class="works__card__subtitle-li fw-600 fz-13 lh-16 clr-n-500">Back End Dev</li>
+						<li class="works__card__subtitle-li fw-600 fz-13 lh-16 clr-n-500">2015</li>
+					</ul>
```
Comment (OLIPLICHE) [inline index.html:106]: - [x] Kindly stick to the design which was provided.
- [x] The original design:
u2

- [x] Your Design: 
u1

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @ONJoseph , 

Good job so far!
There are some issues that you still need to work on to get your project approved, but you are almost there!

## Good Points: 👍
 

- [ ] Linter result is green ✔️
- [ ] node_modules folder is in the .gitignore file and not in the repo✔️
- [ ] Pull Request have a short summary describing introduced changes in more detail✔️

## Required Changes ♻️
- **CHECK THE COMMENTS UNDER THE REVIEW** ⬇.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**You can also consider**:
- Pull Request should have a descriptive title telling you what changes are inside.
- Commit messages should be meaningful

image


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ONJoseph** [comment]
Thanks a lot for the uplifting comments and the detailed recommended 👍
Will proceed to effect the changes ASAP.

File: index.html (hunk 1 above)
Comment (ONJoseph) [reply index.html:69]: Done. Thanks a lot.

File: index.html (hunk 2 above)
Comment (ONJoseph) [reply index.html]: Done.

File: index.html (hunk 3 above)
Comment (ONJoseph) [reply index.html]: Done

File: index.html (hunk 4 above)
Comment (ONJoseph) [reply index.html]: Done

File: index.html (hunk 5 above)
Comment (ONJoseph) [reply index.html]: Done

**Review by Danieldotcomcoder — APPROVED** [review]
Hello @ONJoseph, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 👍 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ONJoseph** [comment]
Thanks for the uplifting review! Cheers 🥂

**Pull request merged**